deepspeed/inference/engine.py

```diff
@@ -6,13 +6,15 @@
 import torch
 import time
 import os
+import deepspeed.utils.nvtx
```

Comment:
@oelayan7 - looks like this is imported but unused, and that's what is causing the formatting failures, could you check and remove this line if so?

Reply:
Seems we may need this merged first?

#5697

Reply:
Yes, you are right. Sorry for the confusion

Reply:
That PR is merged now.